permission_prompt/security_privacy_self_review.md

```diff
@@ -69,27 +75,23 @@ None.
 
 ## 13. How does this specification distinguish between behavior in first-party and third-party contexts?
 
-Third-party iframes are treated distinctly from the first party embedder: even
-if the first party is served from non-public address space, only the third
-party's address space is considered when applying private network request checks
-to requests made by the third-party iframe.
+For permission prompt in particular, it would be restricted to first-party
+embedders in current stage.
 
-Third-party scripts are not treated distinctly. In the case of
-`document.addressSpace`, this could reveal some information about network
-configuration to third parties. There seems to be no particular reason to treat
-third party scripts differently when checking outgoing requests, however.
+We might want to relax the boundary and also allow same-origin child context to
+be able to raise the permission by default. However, third-party iframes would
+be exclude from that.
 
-One area of discussion is
-whether or not to treat sandboxed iframes as `public`: see [#26](https://github.com/WICG/private-network-access/issues/26). It seems a good
-idea to do so, enabling web developers to include third-party content on
-non-public websites without allowing it to poke at non-public resources. On the
-other hand, this might help malicious websites trying to determine which IP
-address they are being accessed from by providing them with a baseline against
-which to compare their own capabilities.
+In the future, it might make sense to allow top-level documents to control its
+permission with permission policy and even allow the private network access
+permission to be delegated to third-party contexts. In that case, third-party
+context should be the one taking into account for the permission.
```

Comment:
```suggestion
In the future, it might make sense to allow top-level documents to control its
permission with permission policy and even allow the private network access
permission to be delegated to third-party contexts. In that case, the third-party
context should be the one taken into account for the permission.
```